homeassistant/const.py

```diff
@@ -315,6 +315,9 @@
 SERVICE_HOMEASSISTANT_STOP = 'stop'
 SERVICE_HOMEASSISTANT_RESTART = 'restart'
 
+SERVICE_ARM = 'arm'
```

Comment:
I didn't see a const.py inside the component/ directory.  Shall I just create a const.py inside components/ directory or shall I just add this inside the __init__.py or something !?